scripts/qa_crowbarsetup.sh

```diff
@@ -1867,6 +1867,14 @@ function enable_ssl_generic
             fi
             return
         ;;
+        database)
+            if iscloudver 7plus && [[ "$want_database_sql_engine" == "mysql" ]] ; then
+                $p "$a['mysql']['ssl']['enabled']" true
+                $p "$a['mysql']['ssl']['generate_certs']" true
+                $p "$a['mysql']['ssl']['insecure']" true
```

Comment:
when you append ['mysql'] to $a, you could drop 2 lines and use the generic codepath after the case for mysql - and only do return in the else case

Reply:
Hm, true, but that would be difference to the other services.

Reply:
Also, the generic path does not set "enabled" bit

Reply:
would be similar to heat, manila and "*" below.

Reply:
See also https://github.com/SUSE-Cloud/automation/pull/2211

Reply:
I would recommend to **not** merge #2211 and go with this PR as it is.
Having the config changes explicitly is more clear and easier to understand than hiding it through manipulating `$a`.